### ISSUE TITLE
Call lua / lua plugins from the command line

### ISSUE THREAD

**Issue by tobiasBora** [xournalpp/xournalpp#6074]
### Is Your Feature Request Related to a Problem?

In order to implement overlay on slides like:
-    slide 1 contains content A
-    slide 2 contains content A + B
-    slide 3 contains content A + C

I was recommended here https://github.com/xournalpp/xournalpp/issues/6062 to write a plugin. Once is solved, I'll be able to run a Lua plugin to export the slides. But this would require me from manually running the export command. Unfortunately, I cannot do that manually as the export is run automatically in my setup by some latex files (using robust-externalize to integrate it in a beamer presentation automatically)… so I need to find a way to run the lua script from commandline.

### Feature Description

It would be great to allow users to run Lua scripts from the command line like:
```
$ xournalpp file.xopp --run-plugin-function nameplugin.function --arg firstArgument --arg secondArg --arg thirdArg
```

### Alternative Solutions

None found.

**Comment by Scratch2xs** [comment]
Yes, this would be great. I want to enhance the xournalpp Obsidian plugin, so it exports with the xournalpp-htr plugin, so my handwritten notes can be indexed and searched in Obsidian
